### PR TITLE
Reduce build time

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,12 +5,10 @@ before_install:
...
+    - sh ./travis_setup.sh -p "rake yast2-devtools yast2 ruby-rspec" -g "yast-rake gettext"
 script:
     - rake check:syntax
     - rake check:pot
-    - make -f Makefile.cvs
-    - make
-    - sudo make install
-    - make check
+    - sudo rake install
+    - rake test:unit check
```
Comment (lslezak) [inline .travis.yml]: NP: the usual workflow is to test first and then install, but if that requires some `$LOAD_PATH` or `$Y2DIR` tweaks I'm fine with that. (And in Travis it does not matter much.)

File: package/yast2-ftp-server.changes [hunk 2]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Sep 20 10:40:22 UTC 2016 - jreidinger@suse.com
+
+- reduce rpm build time (bnc#999203)
```
Comment (lslezak) [inline package/yast2-ftp-server.changes]: `bsc...` :wink:

File: test/ftpserver_test.rb [hunk 3]
```diff
@@ -4,6 +4,16 @@
...
 
+# stub module to prevent its Import
+# Useful for modules from different yast packages, to avoid build dependencies
+def stub_module(name)
+  Yast.const_set name.to_sym, Class.new { def self.fake_method; end }
+end
+
+stub_module("Users")
+
+
```
Comment (lslezak) [inline test/ftpserver_test.rb]: Extra empty lines. What about enabling Rubocop? Could you try it to see how much difficult it would be?

File: .travis.yml (hunk 1 above)
Comment (jreidinger) [inline .travis.yml]: no, not needed, I can switch it.

**Comment by lslezak** [comment]
And Travis fails with `Don't know how to build task 'check'`.

File: test/ftpserver_test.rb (hunk 3 above)
Comment (jreidinger) [inline test/ftpserver_test.rb]: I would like to do it as separate pull request as it is out of scope this PBI and I would like to focus on critical path reduction.

**Comment by jreidinger** [comment]
thanks
